docs/kind.md

```diff
@@ -418,6 +418,16 @@ Once testing is complete, to tear down the KIND deployment:
 $ kind delete cluster --name ovn
 ```
 
+### Using specific Kind container image and tag
```

Comment:
We should probably have some warning here like "you may break everything if you change the node image" :)

Reply:
done. ;)